src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -90,11 +90,20 @@ type Manager struct {
 	// FilePath: required, path to the config file on disk
 	FilePath string
 
-	// EmptyConfig: required, factor for new empty Configs
+	// ValidateConfig: optional. If specified, the config loaded from disk is
+	// validated with this function. If validation fails, the config is replaced
+	// with the result of DefaultConfig.
+	ValidateConfig func(cfg Config) error
+
+	// DefaultConfig: optional. If specified, and there's a problem reading the
+	// config from disk or that , this is used.
+	DefaultConfig func() (Config, error)
```

Comment:
New function that allows application of a default config (in our case, the embedded bootstrap config).